src/ctelnet.cpp

```diff
@@ -1760,7 +1760,7 @@ void cTelnet::processTelnetCommand(const std::string& command)
                      * Once we comply with that we can be certain that Mud
                      * Server encoding will NOT be an issue!
                      */
-                    cmd += termType.toLatin1().data();
+                    cmd += termType.toUtf8().constData();
```

Comment:
See the comment - it is another case where retaining the Latin1 form might highlight that the characters that can be used are restricted...

Reply:
Why would you need to do that? The source may not be Latin1 either. Also, pure UTF-8 is at least legal Telnet (i.e. toUtf8 would result in a legal output no matter which garbage is in `termType`) while some Latin1 characters must be escaped.